.github/workflows/coverage.yml

```diff
@@ -64,10 +72,11 @@ jobs:
           wget https://apt.llvm.org/llvm-snapshot.gpg.key
           sudo apt-key add llvm-snapshot.gpg.key
           rm llvm-snapshot.gpg.key
-          sudo apt-add-repository "deb https://apt.llvm.org/focal/ llvm-toolchain-focal main"
+          VERSION=${CC/#clang-/-}
+          sudo apt-add-repository "deb https://apt.llvm.org/focal/ llvm-toolchain-focal${VERSION} main"
           sudo apt-get update
+          sudo apt-get install $CC gcovr
```

Comment:
`clang++` as a package does not exist, neither as an alias in packages.ubuntu.com for focal, nor on apt.llvm.org. I'm not sure how it worked pre 13, must be an internal alias of GHA. But post 13, since their images don't supply those, apt barks and fails all Linux jobs.

Reply:
Ah, ok.. it used to exist. This is probably part of the restructuring they've been doing and was probably a meta-package for a while without us noticing. No worries then